iteration/iteration_test.go

```diff
@@ -128,3 +128,46 @@ func (test *TestIterationRepository) TestListIterationBySpace() {
 	assert.Nil(t, err)
 	assert.Len(t, its, 3)
 }
+
+func (test *TestIterationRepository) TestUpdateIteration() {
+	t := test.T()
+	resource.Require(t, resource.Database)
+
+	repo := iteration.NewIterationRepository(test.DB)
+
+	start := time.Now()
+	end := start.Add(time.Hour * (24 * 8 * 3))
+	name := "Sprint #24"
+
+	i := iteration.Iteration{
+		Name:    name,
+		SpaceID: uuid.NewV4(),
+		StartAt: &start,
+		EndAt:   &end,
+	}
+	// creates an iteration
+	repo.Create(context.Background(), &i)
+	if i.ID == uuid.Nil {
+		t.Errorf("Comment was not created, ID nil")
+	}
+
+	desc := "Updated item"
+	i.Description = &desc
+	updatedName := "Sprint 25"
+	i.Name = updatedName
+	// update iteration with new values of Name and Desc
+	updatedIteration, err := repo.Save(context.Background(), i)
+	assert.Nil(t, err)
```

Comment:
Yes I remember your comment on using `Nil` from `require` vs `assert`
done in https://github.com/almighty/almighty-core/pull/652/commits/bcc81828a5d13cc92cd49634172d8506296fd4ff 
Thanks.